### PR TITLE
I keep my IDE open and grep for every spelling, punctuation, or grammar error I can find: Part 2

### LINKED ISSUE #6909: Deathnettle examine spacing
image

The **glowingnettle incites _ragein you just from looking at it!_**

### PR DESCRIPTION
**Pull request by 9600bauds**
Fixes some large items, such as mech parts, showing "it is a  item" in their description
Fixes #6909
Fixes HTML in mousetrap box description
Fixes arguments in many messages that were intended to not be visible to blind people
Renames mech "Syringe Gun" to "Exosuit-Mounted Syringe Gun", to differentiate it a little better, since they have the same icon and almost the same name as the non-mech deal.

### CODE REVIEW

File: code/modules/mob/living/silicon/pai/software.dm [hunk 1]
```diff
@@ -38,7 +38,7 @@
 	if(temp)
 		left_part = temp
 	else if(src.stat == 2)						// Show some flavor text if the pAI is dead
-		left_part = "<b><font color=red>�Rr�R �a�� ��Rr����o�</font></b>"
+		left_part = "<b><font color=red>�Rr�R �a�� ��Rr����o�</font></b>"
```
Comment (9600bauds) [inline code/modules/mob/living/silicon/pai/software.dm]: Fucking christ I removed this out of smartgit at least 5 times

**Comment by Exxion** [comment]
> Part 2

Isn't this part 4 or so?

**Comment by 9600bauds** [comment]
sshh

**Comment by ihadtoregisterforthis** [comment]
```
/obj/item/weapon/SWF_uplink
    name = "station-bounced radio"
    desc = "used to comunicate it appears."
```

The description ain't capitalized

**Comment by ihadtoregisterforthis** [comment]
Found a jumpsuit called **stilsuit**, that should be _sti**ll**suit_ but is sometimes called **dtilsuit**, in the chameleon/all suit at least.

**Comment by 9600bauds** [comment]
literally how

**Pull request merged**